youtube_dl_gui/locale/es_ES/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -219,12 +216,12 @@ msgstr "Estado"
 
 #: youtube_dl_gui/mainframe.py:235
 msgid "Get URL"
-msgstr ""
+msgstr "Buscar dirección URL"
```

Comment:
This translates to "Search URL" which does not match with the original translation.
Are you sure this is correct?